FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_DNS.c

```diff
@@ -891,12 +1265,13 @@ static const DNSMessage_t xDefaultPartDNSHeader =
 
 	/* Finish off the record. Cast the record onto DNSTail_t stucture to easily
 	 * access the fields of the DNS Message. */
-	pxTail = ipCAST_PTR_TO_TYPE_PTR( DNSTail_t, &( pucByte[ 1 ] ) );
+	pxTail = ipPOINTER_CAST(DNSTail_t *, &( pucByte[ 1 ] ) );
```

Comment:
Instance.

Reply:
Ah, that is what you mean with "instance" :-) Understood.
Done